geospaas/viewer/templates/viewer/base.html

```diff
@@ -1,5 +1,5 @@
 {% extends "viewer/site_base.html" %}
-{% load staticfiles %}
+{% load static %}
```

Comment:
Does it still work with Django 2.2?

Reply:
I have built it and it is working on my local machine